testsetup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Remove this file. Anything you need this for means that the tests you're writing aren't thorough enough.